compass/ocean/tests/tides/mesh/vr45to5/__init__.py

```diff
@@ -0,0 +1,284 @@
+import os
+
+import jigsawpy
+import netCDF4 as nc
+import numpy as np
+from mpas_tools.cime.constants import constants
+from mpas_tools.logging import check_call
+from skimage.filters import gaussian, scharr
+
+from compass.mesh.spherical import SphericalBaseStep
+
+# from skimage.filters import farid
+# from skimage.filters.rank import median, percentile
+
+
+class VRTidesMesh(SphericalBaseStep):
```

Comment:
This should probably inherit from `QuasiUniformSphericalMeshStep` then I can reuse the `setup` and `run` methods (if I refactor the return variables from `build_cell_width_lat_lon` and the input variables to `make_jigsaw_mesh`

Reply:
If possible, that does seem better.  But we would want to avoid updating `QuasiUniformSphericalMeshStep` if it can be avoided.